### ISSUE TITLE
Add LBGAT

### ISSUE THREAD

**Issue by jiequancui** [fra31/auto-attack#34]
**Paper**: {Learnable Boundary Guided Adversarial Training; https://arxiv.org/pdf/2011.11164.pdf}

**Venue**: {if applicable, the venue where the paper appeared}

**Dataset and threat model**: {CIFAR-100, L-inf and epsilon 0.031}

**Code**: { [code](https://github.com/FPNAS/LBGAT)}

**Pre-trained model**: [LBGAT0-wideresnet-34-10](https://mycuhk-my.sharepoint.com/:u:/g/personal/1155116769_link_cuhk_edu_hk/Edce7FP6qe1Lm2FwkhOpO6kB6i-3Zstd2IrE-14BX66nyQ?e=6j5MM0), [LBGAT6-wideresnet-34-10](https://mycuhk-my.sharepoint.com/:u:/g/personal/1155116769_link_cuhk_edu_hk/EYFj964RVGtAu9qhhgjGfXwB3H1U0QLE00P_FLPMAfzdYA?e=utPhnz), [LBGAT6-wideresnet-34-20](https://mycuhk-my.sharepoint.com/:u:/r/personal/1155116769_link_cuhk_edu_hk/Documents/PhD_projects/Fininshed_Project/Archive2020_LBGAT/cifar100_lbgat6_wideresnet34-20.pt?csf=1&web=1&e=DMcYnN)

**Log file**: {link to log file of the evaluation}

**Additional data**: {no}

**Clean and robust accuracy**: {70.25/27.16, 60.64/29.33, 62.55/30.20}

**Architecture**: {wideresnet-34-10; wideresnet-34-20}

**Description of the model/defense**: {Our method aims to enhance model robustness while preserving high natural accuracy. }

**Comment by FPNAS** [comment]
Google Drive links for our models:

[LBGAT0-wideresnet-34-10](https://drive.google.com/file/d/1CijxcgW1U8yfrB3n4dyxUbcotVaxyZyA/view?usp=sharing), [LBGAT6-wideresnet-34-10](https://drive.google.com/file/d/1pzheoiTtoh0qKWcyjFwwxFK6GL0yXAQI/view?usp=sharing), [LBGAT6-wideresnet-34-20](https://drive.google.com/file/d/18iond836snl_chrBL0s7f_BY-5AUWfej/view?usp=sharing)

**Comment by fra31** [comment]
Hi,

thanks for the submission. I could reproduce the results and I'm happy to add the modes to list.

**Comment by FPNAS** [comment]
Thank you very much

**Comment by fra31** [comment]
Added.

**Comment by jiequancui** [comment]
Hi,

Our paper "Learnable boundary guided adversarial training" is to appear in ICCV2021. Can you help update the "Venue status" for our LBGAT models in Robust Bench?

**Comment by fra31** [comment]
Hi,

sure, I can update it.

**Comment by fra31** [comment]
It should be updated. Congrats for the acceptance!

**Comment by jiequancui** [comment]
Big thanks.
